src/test/resources/solr/collection1/conf/schema.xml

```diff
@@ -17,5 +17,6 @@
 
 	<field name="title"     type="string" indexed="true"  stored="true" />
 	<field name="access"    type="string" indexed="false" stored="true" default="c:0" docValues="true" />
+	<field name="accesses"  type="string" indexed="false" stored="true" default="c:0" docValues="true" multiValued="true" />
```

Comment:
I  would do it, as you proposed in EXT:solr to just change the "multiValued" property there. I did it here like this to keep it backwards compatible.